src/Core/src/Platform/Android/LabelExtensions.cs

```diff
@@ -22,5 +23,63 @@ public static void UpdateTextColor(this TextView textView, ILabel label,  Maui.C
 				textView.SetTextColor(textColor.ToNative());
 			}				
 		}
-	}
+
+        public static void UpdateLineBreakMode(this TextView textView, ILabel label)
+        {
+            textView.SetLineBreakMode(label);
+        }
+
+        public static void UpdateMaxLines(this TextView textView, ILabel label)
+        {
+            var maxLines = label.MaxLines;
+
+            if (maxLines == 0)
+            {
+                // MaxLines is not explicitly set, so just let it be whatever gets set by LineBreakMode
+                textView.SetLineBreakMode(label);
+                return;
+            }
+
+            textView.SetMaxLines(maxLines);
+        }
+
+        internal static void SetLineBreakMode(this TextView textView, ILabel label)
+        {
+            var lineBreakMode = label.LineBreakMode;
+
+            int maxLines = int.MaxValue;
```

Comment:
As noted in a comment above, this will always override this previously set max lines. Changing the break mode resets the maxLines.